src/components/organisms/Room/VideoErrorModal.tsx

```diff
@@ -29,7 +30,7 @@ export const VideoErrorModal: FC<PropsType> = ({
   onRetry,
 }) => {
   return (
-    <Modal show={show} onHide={onHide} backdrop="static">
+    <Modal isOpen={show} onClose={onHide} shouldCloseOnOverlayClick={false}>
```

Comment:
`shouldCloseOnOverlayClick`?

maybe `autohide` or `autoclose` is enough and if `true` is the more common use, then maybe a prop with other name for the opposite case `noAutohide` or `noAutoClose`

Reply:
overlayClose now ead67ab